.vscode/launch.json

```diff
@@ -0,0 +1,44 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Mocha Current Spec File",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "program": "${workspaceFolder}/node_modules/.bin/mochapack",
+      "args": [
+        "--webpack-config", "config/webpack/test.js",
+        "--require", "spec/javascript/setup.js",
+        "--timeout", "999999",
+        "--colors",
+        "${file}"
+      ],
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Mocha All JS Specs",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "program": "${workspaceFolder}/node_modules/.bin/mochapack",
+      "args": [
+        "--webpack-config", "config/webpack/test.js",
+        "--require", "spec/javascript/setup.js",
+        "--timeout", "999999",
+        "--colors",
+        "spec/javascript/**/*.spec.js"
```

Comment:
I am looking forward to https://github.com/sysgears/mochapack/pull/63 being resolved so that this file can reference a script in package.json instead of the args here potentially getting out of sync